packages/components/src/Spinner/src/Spinner.module.css

```diff
@@ -0,0 +1,83 @@
+@keyframes spin {
+    0% {
+        transform: rotate(0deg);
+    }
+
+    100% {
+        transform: rotate(360deg);
+    }
+}
+
+.hop-spinner {
+    --hop-spinner-component-border-radius: var(--hop-shape-circle);
+    --hop-spinner-component-sm-wheel-size: 1rem;
```

Comment:
The dimensions of the spinner seem strange to me, but I can't find any reference to this component in Figma. 

```
small: 24x16px
medium: 28x20px
large: 32x24px
```

Reply:
```
.hop-spinner--sm {
    --width: var(--hop-spinner-component-sm-wheel-size);
    --height: var(--hop-spinner-component-sm-wheel-size);
}
```

The width and height are set to the same value, so it's 
```
small: 16x16px
medium: 20x20px 
large: 24x24px
```

There is indeed no figma for this component yet

Reply:
If you look in the browser at the element containing the spinner and label, you get 28x20px, which you'd expect to be 20x20px. 

<img width="1238" alt="Capture d’écran, le 2024-03-13 à 09 34 06" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/a1b22112-7a9f-4e3f-9ec8-83d4d04d03ac">

Reply:
looks like it's because he has column-gap on that element, the child. If he removes that, it's fine.